sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -222,7 +219,6 @@ def read_item(
                 "the populate_query_metrics flag does not apply to this method and will be removed in the future",
                 UserWarning,
             )
-            request_options["populateQueryMetrics"] = populate_query_metrics
```

Comment:
looks like we're removing it. Should we update the user warning above?

Reply:
Removing it shouldn't change anything to the behaviour of the method.

Reply:
I've reverted these for now to avoid conflating it with the typing updates.